integration_tests/suite/test_call_log_generation.py

```diff
@@ -262,6 +262,169 @@ def call_log_is_missing_source_user():
+ CHAN_START                     | 2022-07-22 19:18:47.630038        | Harry Potter          | 1603          |           | 1900          | mycontext     | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     |
+ WAZO_CONFERENCE                | 2022-07-22 19:18:47.667896        | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     | {"extra":" NAME: myconference"}
+ ANSWER                         | 2022-07-22 19:18:47.671298        | Harry Potter          | 1603          | 1603      | pickup        | xivo-pickup   | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     |
+ WAZO_CALL_LOG_DESTINATION      | 2022-07-22 19:18:48.698413        | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     | {"extra":"type: conference,id: 1"}
+ BRIDGE_ENTER                   | 2022-07-22 19:18:52.804353        | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     | {"bridge_id":"1f633aec-e2f9-4910-a8bb-0601439fab15","bridge_technology":"softmix"}
+ BRIDGE_EXIT                    | 2022-07-22 19:18:53.150128        | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     | {"bridge_id":"1f633aec-e2f9-4910-a8bb-0601439fab15","bridge_technology":"softmix"}
+ HANGUP                         | 2022-07-22 19:18:53.54901         | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     | {"hangupcause":16,"hangupsource":"PJSIP/cul113qn-00000004","dialstatus":""}
+ CHAN_END                       | 2022-07-22 19:18:53.54901         | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     |
+ LINKEDID_END                   | 2022-07-22 19:18:53.54901         | Harry Potter          | 1603          | 1603      | s             | conference    | PJSIP/cul113qn-00000004   | 1658531927.10 | 1658531927.10     |
+'''
+    )
+    def test_conference_has_destination_details_setup_correctly(self):
+        conference_id = '1'
+        self._assert_last_call_log_matches(
+            '1658531927.10',
+            has_properties(
+                destination_details=contains_inanyorder(
+                    has_properties(
+                        destination_details_key='type',
+                        destination_details_value='conference',
+                    ),
+                    has_properties(
+                        destination_details_key='conference_id',
+                        destination_details_value=conference_id,
```

Comment:
Could you make the `conference_id` value as an integer?

Reply:
Sure I will make it for Marshmallow schema and `test_cdr.py`